pkg/cmd/clustertriggerbinding/list.go

```diff
@@ -98,7 +103,7 @@ or
 	}
 
 	f.AddFlags(c)
-
+	c.Flags().BoolVarP(&opts.NoHeaders, "no-headers", "", opts.NoHeaders, "do not print column headers with output (default print column headers with output)")
```

Comment:
cant we have a default value false

Reply:
The default value is false only. Zero value of `opts.NoHeaders` is false. So it's same to right `false` or `opts.NoHeaders`.